docs/antora/modules/command-line/pages/command_line_tools.adoc

```diff
@@ -0,0 +1,376 @@
+= Command Line Tools
+keywords: queue, java, command line,
+author: Kevin Powe
+:reftext: Command Line Tools
+:navtitle: Comamnd Line Tools
+:source-highlighter: highlight.js
+
+== Overview
+
+The following command line tools can be used to interact with Chronicle Queue.
+
+[cols="1,1"]
+|===
+
+| ★ <<_archiverollfiles,*ArchiveRollFiles*>>
+| Assists in management of roll cycle files
+
+| <<_chroniclehistoryreadermain,*ChronicleHistoryReaderMain*>>
+| Reads history from queue and outputs latency histograms
+
+| <<_chroniclereadermain,*ChronicleReaderMain*>>
+| Displays records from a queue in text form, supporting more complex operations
+
+| <<_chroniclewritermain,*ChronicleWriterMain*>>
+| Write contents of files provided to queue
+
+| <<_dumpmain,*DumpMain*>>
```

Comment:
I would put this last as it is a super low level debugging tool

Reply:
I hear you - I think there's a case though for keeping alphabetical order for consistency. I'll add a brief note at the start of the entry pointing folks to ChronicleReaderMain